agent_build_refactored/tools/runner.py

```diff
@@ -15,31 +15,32 @@
 
```

Comment:
This diff is quite messy, but its mainly just refactored in order to be able to use remote docker engine form ec2 to run some steps in it.